src/server/dfly_main.cc

```diff
@@ -611,9 +611,11 @@ void sigill_hdlr(int signo) {
 }
 
 void PrintBasicUsageInfo() {
+  auto ends_with_slash = [](const string_view v) { return v.back() == '/'; };
   std::cout << "* Logs will be written to the first available of the following paths:\n";
   for (const auto& dir : google::GetLoggingDirectories()) {
-    std::cout << dir << "dragonfly.*\n";
+    const string maybe_slash = ends_with_slash(dir) ? "" : "/";
```

Comment:
I'd use `absl::EndsWith()` here:
https://github.com/abseil/abseil-cpp/blob/master/absl/strings/match.h#L65
It also gracefully handles the case of an empty string, which the existing code doesn't really. I guess one could argue that the terminating null would work here, but `string_view`s don't necessarily have these, so at least in debug it would alert on that.

Reply:
wow nice, I should have looked on `absl`, ofc it has `EndsWith` :D  (which C++20 has as well ;))